app/agents/voice/automatic/services/mcp/automatic_client.py

```
@@ -18,20 +18,89 @@
+        logger.info(f"MCP Client initialized: shopId='{self._shop_id}', isNeurolink={self._is_neurolink}, serverUrl='{server_url}'")
+        
+        if self._is_neurolink:
+            logger.info(f"Using Neurolink server for shop_id: {self._shop_id}")
+            logger.info(f"MCP Client Context being sent to neurolink: {json.dumps(context, indent=2)}")
+            logger.debug(f"Base64 encoded x-context header: {self._context_b64}")
+            
+            # Decode and show what the server will receive
+            try:
+                decoded_context = json.loads(base64.b64decode(self._context_b64).decode())
+                logger.info(f"Decoded context (what server receives): {json.dumps(decoded_context, indent=2)}")
+            except Exception as e:
```

Ruff (0.12.2)

Do not catch blind exception: `Exception`

(BLE001)